src/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -1438,6 +1441,18 @@ private static void TestExportStore(X509ContentType ct)
             }
         }
 
+        public static IEnumerable<object[]> StorageFlags
```

Comment:
Those tests already exist, `EphemeralImport_HasNoKeyName` and friends.  Those show that there's no notion of persistence for the key.